inference-engine/src/mkldnn_plugin/nodes/mkldnn_cum_sum_node.cpp

```diff
@@ -95,43 +95,17 @@ void MKLDNNCumSumNode::execute(mkldnn::stream strm) {
     if (inputShapes.size() == numOfInputs)
         axis = getAxis(getParentEdgeAt(AXIS)->getMemory(), getParentEdgeAt(CUM_SUM_DATA)->getMemory());
 
-    switch (dataPrecision) {
-        case Precision::I8   : {
-            exec<int8_t>();
-            break;
-        }
-        case Precision::U8   : {
-            exec<uint8_t>();
-            break;
-        }
-        case Precision::I16  : {
-            exec<int16_t>();
-            break;
-        }
-        case Precision::I32  : {
-            exec<int32_t>();
-            break;
-        }
-        case Precision::FP32 : {
-            exec<float>();
-            break;
-        }
-        case Precision::I64  : {
-            exec<int64_t>();
-            break;
-        }
-        case Precision::U64  : {
-            exec<uint64_t>();
-            break;
-        }
-        default : {
-            std::string errorMsg = errorPrefix + " has unsupported 'data' input precision: " + dataPrecision.name();
-            IE_THROW() << errorMsg;
-        }
-    }
+    OV_SWITCH(MKLDNNPlugin, CumSumEmitter, this, dataPrecision,
```

Comment:
Why do we call this differently across the nodes?
Some nodes call this "Executor"

Reply:
Discussed offline. Renamed as "Execute" because "Executor" is about caching. Done, thanks!